src/cards/cardIdentifier.ts

```diff
@@ -0,0 +1,31 @@
+export default class CardIdentifier {
+
+  public static official(id: number) {
+    return new CardIdentifier('official', id);
+  }
+
+  public static satireGaming(id: number) {
+    return new CardIdentifier('satire-gaming', id);
+  }
+
+  constructor(readonly origin: string, readonly id: number) {
```

Comment:
Maybe `id` is not the best name here because it's not unique (anymore). Maybe `n` or `cardNumber` or similar?

Reply:
or `subId`?

Reply:
I like `cardId` better. With `subId` something like `123-456` comes to my mind. `456` is the subId.